### PR TITLE
WIP saves to db

### PR DESCRIPTION
**Pull request by neimanchen**

### CODE REVIEW

File: src/client/components/AddItem/AddItem.jsx [hunk 1]
```diff
@@ -11,6 +14,16 @@ export class AddItem extends React.Component {
     super(props);
   }
 
+  componentDidMount() {
+    Axios.get('/uploaditeminfo')
+    .then((response) => {
+      this.props.actions.updateStyles(response.data);
+    })
+    .catch((err) => {
+      this.props.actions.updateStyles(err);
```
Comment (skoshkarli) [inline src/client/components/AddItem/AddItem.jsx]: any reason for storing the error in store?

File: src/client/components/AddItem/UploadForm.jsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 import { Field, reduxForm } from 'redux-form';
 import { withRouter } from 'react-router-dom';
-import { Form, Button } from 'semantic-ui-react';
+import { Form, Button, Dropdown } from 'semantic-ui-react';
 
 const UploadForm = (props) => {
-  const { handleSubmit, pristine, reset, submitting } = props;
-
+  const { pristine, submitting } = props;
+  const sizes = ['XS', 'S', 'M', 'L', 'XL', 'XXL'];
```
Comment (skoshkarli) [inline src/client/components/AddItem/UploadForm.jsx]: @hcnimi do we have sizes in our db somewhere? i dont think so but i thought i'd double check

File: src/client/components/AddItem/UploadItemInfo.jsx [hunk 3]
```diff
@@ -5,14 +5,44 @@ import { connect } from 'react-redux';
...
+        description: this.props.formStates.description,
+        size: this.props.formStates.size,
+        url: this.props.imageURL,
+        price: this.props.formStates.price,
+        category: this.props.formStates.category,
+        color: this.props.formStates.color,
+        date: new Date(this.props.formStates.date)
+      }
+    })
+    console.log('item', item)
```
Comment (skoshkarli) [inline src/client/components/AddItem/UploadItemInfo.jsx]: console.log

File: src/client/components/AddItem/AddItem.jsx (hunk 1 above)
Comment (neimanchen) [reply src/client/components/AddItem/AddItem.jsx]: temporary solution

File: src/client/components/AddItem/UploadItemInfo.jsx [hunk 4]
```diff
@@ -3,16 +3,46 @@ import { Grid, Modal, Button, Header, Image, Form } from 'semantic-ui-react';
...
+        size: this.props.formStates.size,
+        url: this.props.imageURL,
+        price: this.props.formStates.price,
+        category: this.props.formStates.category,
+        color: this.props.formStates.color,
+        date: new Date(this.props.formStates.date)
+      }
+    })
+    .catch((err) => {
+      console.log(err)
```
Comment (skoshkarli) [inline src/client/components/AddItem/UploadItemInfo.jsx]: console.log

File: src/client/actions/addItemActions.js [hunk 5]
```diff
@@ -60,3 +77,10 @@ export function uploadFail(error) {
     error
```
Comment (hcnimi) [inline src/client/actions/addItemActions.js]: shouldn't this be "data" and not {data}?

Comment (neimanchen) [reply src/client/actions/addItemActions.js]: you're right

File: src/client/components/AddItem/UploadForm.jsx (hunk 2 above)
Comment (hcnimi) [reply src/client/components/AddItem/UploadForm.jsx]: we don't.

**Pull request merged**